identity/signer_test.go

```diff
@@ -0,0 +1,23 @@
+package identity
+
+import (
+	"testing"
+	"github.com/stretchr/testify/assert"
+	"encoding/hex"
+)
+
+func TestSigning(t *testing.T) {
+	ks := getIdentityTestKeystore()
+	identity := FromAddress("0x1e35193c8cadAA15b43B05ae3D882C91F49BB0Aa")
+	manager := NewIdentityManager(ks)
+	manager.Unlock(identity, "test_passphrase")
+
+	signer := manager.GetSigner(identity)
```

Comment:
You dont need full manager to test only `signer.Sign()`

Reply:
Accounts still need to be unlocked. How would you approach this?

Reply:
Usualy tests skip regular factory `New..`.
And loads just dependencies which are tested:
```
keystore := getIdentityTestKeystore()
keystore.Unlock("0x1e35193c8cadAA15b43B05ae3D882C91F49BB0Aa")

signer := keystoreSigner{keystore: keystore}
```

Reply:
Fixed